bin/loklakinit.py

```diff
@@ -9,151 +10,187 @@
 def sigint_handler(signum, frame):
   print "\nloklakinit cancelled"
   exit(0)
- 
-signal.signal(signal.SIGINT, sigint_handler)
 
-print "This utility will walk you through creating a app.json file.\n\
-It only covers the most common items, and tries to guess sensible defaults.\n\
-\n\
-After collecting all the necessary parameters a sample application directory\n\
-structure will be created along with sample files.\n\
-\n\
-Press ^C at any time to quit.\n"
+if __name__ == "__main__" :
+  signal.signal(signal.SIGINT, sigint_handler)
+
+  months = ["january",
+  "February",
+  "March",
+  "April",
+  "May",
+  "Jun",
+  "July",
+  "August",
+  "September",
+  "October",
+  "November",
+  "December"]
+
+  print "This utility will walk you through creating a app.json file.\n\
+  It only covers the most common items, and tries to guess sensible defaults.\n\
+  \n\
+  After collecting all the necessary parameters a sample application directory\n\
+  structure will be created along with sample files.\n\
+  \n\
+  Press ^C at any time to quit.\n"
+
+  app_json = collections.OrderedDict()
+
+  app_json["@context"] = "http://schema.org"
+  app_json["@type"] = "SoftwareApplication"
+  app_json["permissions"] = "/api/search.json"
+  app_json["name"] = "myloklakapp"
+  app_json["headline"] = "My first loklak app"
+  app_json["alternativeHeadline"] = app_json["headline"]
+  app_json["applicationCategory"] = "Misc"
+  app_json["applicationSubCategory"] = ""
+  app_json["operatingSystem"] = "http://loklak.org"
+  app_json["promoImage"] = "promo.png"
+  app_json["appImages"] = ""
+  app_json["oneLineDescription"] = ""
+  app_json["getStarted"] = "getStarted.md"
+  app_json["appUse"] = "appUse.md"
+  app_json["others"] = "others.md"
+  app_json["appSource"] = ""
+  app_json["contributors"] = ""
+  app_json["techStack"] = ""
+  app_json["license"] = {"name": "LGPL 2.1", "url": "https://www.gnu.org/licenses/old-licenses/lgpl-2.1"}
+  app_json["version"] = "1.0"
+  app_json["updated"] = ""
+
+  author = collections.OrderedDict()
+  author["@type"] = "Person"
+  author["name"] = ""
+  author["email"] = ""
+  author["url"] = ""
+  author["sameAs"] = ""
 
-app_json = collections.OrderedDict()
+  app_json["author"] = author
 
-app_json["@context"] = "http://schema.org"
-app_json["@type"] = "SoftwareApplication"
-app_json["permissions"] = "/api/search.json"
-app_json["name"] = "myloklakapp"
-app_json["headline"] = "My first loklak app"
-app_json["alternativeHeadline"] = app_json["headline"]
-app_json["applicationCategory"] = "Misc"
-app_json["applicationSubCategory"] = ""
-app_json["operatingSystem"] = "http://loklak.org"
-app_json["promoImage"] = "promo.png"
-app_json["appImages"] = ""
-app_json["oneLineDescription"] = ""
-app_json["getStarted"] = "getStarted.md"
-app_json["appUse"] = "appUse.md"
-app_json["others"] = "others.md"
+  while True :
```

Comment:
Rather than taking for user input like this (explicitly asking for each *key*), can a function be created that takes an `OrderedDict` as parameter and iterates through its *keys* for taking user input?
The function is called two times once for `app_json` and once for `author`, once the user provides input for both of them, and confirms with `yes` then `author` is assigned to `app_json`.
@djmgit Whats your thought on this?